blue_controller_manager/src/blue_controller_manager.cpp

```diff
@@ -51,12 +55,23 @@ int main(int argc, char** argv)
     ros::Time current_time = ros::Time::now();
     cm.update(current_time, current_time - prev_time);
     double frequency = 1.0 / (current_time - prev_time).toSec();
+    sum_freq += frequency;
     prev_time = current_time;
 
     robot.write();
 
     // Information for user!
-    ROS_INFO_THROTTLE(1.0, "Communication frequency is %.2fHz", frequency);
+    if ((current_time - last_update).toSec() > 1.0) {
+      long iters = count - last_count;
+      double error_perc = (double)(num_errors) / (double)(iters) * 100.0;
+      double avg_freq = sum_freq / (double)(iters);
```

Comment:
this could be simplified a little?

we could just do `iters / time elapsed` in this if statement (or even just `iters`) instead of computing the `frequency` value up top at every cycle

Reply:
Yeah, good point.

Reply:
Updated with commit 11d8f23